src/framework/audio/CMakeLists.txt

```diff
@@ -277,23 +282,22 @@ endif()
 
 if (MUSE_MODULE_AUDIO_JACK)
     find_package(Jack REQUIRED)
-    set(MODULE_INCLUDE_PRIVATE ${MODULE_INCLUDE_PRIVATE} ${JACK_INCLUDE_DIRS} )
-    set(MODULE_LINK ${MODULE_LINK} ${JACK_LIBRARIES} pthread )
+    set(MODULE_INCLUDE_PRIVATE ${MODULE_INCLUDE_PRIVATE} ${JACK_INCLUDE_DIRS})
+    set(MODULE_LINK ${MODULE_LINK} ${JACK_LDFLAGS} pthread)
```

Comment:
Often, a `find_package` also generates an imported target, probably called something like `Jack::Jack`. If that is the case, it would be enough to `list(APPEND MODULE_LINK Jack::Jack)`, and that would set up include directories automatically. Might be worth trying.